src/native/third_party/libutp/utp_internal.cpp

```diff
@@ -1975,7 +1975,7 @@ size_t utp_process_incoming(UTPSocket *conn, const byte *packet, size_t len, boo
 		if (pkt == 0 || pkt->transmissions == 0) continue;
 		assert((int)(pkt->payload) >= 0);
 		acked_bytes += pkt->payload;
-		if (conn->mtu_probe_seq && seq == conn->mtu_probe_seq) {
+		if (conn->mtu_probe_seq && seq == static_cast<int>(conn->mtu_probe_seq)) {
```

Comment:
same as isa-l, if this is an upstream project that we use, we first need to make the PR there

Reply:
Thanks, removing the changes.